packages/steiger-plugin-fsd/package.json

```diff
@@ -43,6 +43,7 @@
     "@feature-sliced/filesystem": "^3.0.1",
     "fastest-levenshtein": "^1.0.16",
     "lodash-es": "^4.17.21",
+    "oxc-parser": "^0.47.1",
```

Comment:
praise: kudos for using the oxc-parser :) I was hoping that we would switch to it entirely at some point and drop `precinct` because it isn't flexible enough and doesn't extend well